src/rmodels.c

```diff
@@ -5226,17 +5226,19 @@ static Model LoadGLTF(const char *fileName)
     ***********************************************************************************************/
 
     // Macro to simplify attributes loading code
-    #define LOAD_ATTRIBUTE(accesor, numComp, dataType, dstPtr) \
+    #define LOAD_ATTRIBUTE(accesor, numComp, srcType, dstPtr) LOAD_ATTRIBUTE_CAST(accesor, numComp, srcType, dstPtr, srcType) 
+
+    #define LOAD_ATTRIBUTE_CAST(accesor, numComp, srcType, dstPtr, dstType) \
```

Comment:
@raysan5 As mention in discord this would have to change to something like  

```
if (dstPtr == NULL) { 
  dstPtr = RL_ALLOC(accessor->count * numComp * sizeof(dstType); 
}
```
This would reduce a lot of the 4-6 line `if/else` branches to 1 Line 
```
LOAD_ATTRIBUTE(accessor, numpComp, srcType, dstPtr, dstType)
```